Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby
+MAINTAINER Jack M <exegete@mac.com>
```

Comment:
I am, but I have no issues changing that to an official maintainer.  It's just habit to put myself in.